product-service/initial.sql

```diff
@@ -0,0 +1,19 @@
+insert into products (title, description, price) values
+('ProductOne', 'Short Product Description1', 10),
+('ProductNew', 'Short Product Description3', 12),
+('Product', 'Short Product Description7', 14),
+('ProductTest', 'Short Product Description4', 18),
+('Product2', 'Short Product Description1', 13),
+('ProductName', 'Short Product Description8', 15),
+('Product', 'Short Product Description10', 13),
+('Product', 'Short Product Description', 15)
+
+insert into stocks (product_id, count) values
```

Comment:
This ids will not match with products ids by default

Reply:
agreed, but didn't find the way to use actual ids